### DISCUSSION TITLE
v3.0 Compatibility

### DISCUSSION THREAD

**Discussion by galago** [tailwindlabs/tailwindcss#6350] [Help]
I'm excited to upgrade my projects to v3 but I found some errors with the _typography_ plugin upgrading. Do you have a plan to make it compatible with the new version? If so what's the timeline? 

Thanks for the great effort.

**Comment by NovusOpus** [comment]
You can install the Alpha Version that supports Tailwind CSS V3 with:
`npm install -D @tailwindcss/typography@next`

Source and more info: [https://github.com/tailwindlabs/tailwindcss/discussions/6012](https://github.com/tailwindlabs/tailwindcss/discussions/6012)
